operator/src/main/java/org/bf2/operator/operands/AbstractKafkaCluster.java

```diff
@@ -412,6 +413,11 @@ protected List<GenericKafkaListener> buildListeners(ManagedKafka managedKafka, i
                                 .withName("sre")
                                 .withPort(9096)
                                 .withType(KafkaListenerType.INTERNAL)
+                                .withNetworkPolicyPeers(new NetworkPolicyPeerBuilder()
+                                        .withIpBlock(new IPBlockBuilder()
+                                                .withCidr("0.0.0.0/32")
```

Comment:
It looks like it will block all IP addresses, how is 127.0.0.1 is open for 9096? is that implicit behavior?

Reply:
This applies to traffic from outside the Pod via the Service. Loopback/local connections to 127.0.0.1 won't be affected.